task2.js

```diff
@@ -0,0 +1,34 @@
+const csvFilePath = './csv/nodejs-hw1-ex1.csv'
+const txtFilePath = './out.txt'
+import csv from 'csvtojson'
+import fs from 'fs'
+
+const readStream = fs.createReadStream(csvFilePath)
+const writeStream = fs.createWriteStream(txtFilePath, 'utf8')
+
+const handleError = (error) => {
+    readStream.destroy()
+    csv().destroy()
+    console.error(error.message)
+}
+
+readStream
+    .on('error', (error) => handleError(error))
+    .pipe(csv({
+        colParser: {
+            "Amount": "omit",
+            "Price": "number",
+        },
+        checkType: true
+    }))
+    .on('error', (error) => handleError(error))
+    .subscribe((json) => {
+        for(const [key] of Object.entries(json)){
+            const oldKey = key
+            const newKey = oldKey.toLowerCase()
+            json[newKey] = json[oldKey]
```

Comment:
```Object.entries()``` gives you key and value paired in to array ([link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries)).

```Object.entries(object1).forEach(([key, value]) => { /* do something */})```